src/transformers/utils/import_utils.py

```diff
@@ -747,13 +747,23 @@ def get_major_and_minor_from_version(full_version):
 
 @lru_cache
 def is_torch_xpu_available(check_device=False):
-    "Checks if `intel_extension_for_pytorch` is installed and potentially if a XPU is in the environment"
-    if not is_ipex_available():
+    """
+    Checks if XPU acceleration is available either via `intel_extension_for_pytorch` or
+    via stock PyTorch (>=2.4) and potentially if a XPU is in the environment
+    """
+    if not is_torch_available():
         return False
 
-    import intel_extension_for_pytorch  # noqa: F401
     import torch
 
+    if is_ipex_available():
+        import intel_extension_for_pytorch  # noqa: F401
+    elif not is_accelerate_available("0.32.0.dev"):
```

Comment:
@muellerzr : I added this check, since python version comparison evaluates `0.32.0.dev0 >= 0.32.0` as `False`, so I compared with `0.32.0.dev` and I am not sure that this is correct way. Please, advice.

Reply:
and... I obviously can't raise exception here since all the tests will fail in this case. Problem is that returning `False` here does not give much clarity to the end user - all fails at runtime with pretty the same error as with having wrong accelerate.